### ISSUE TITLE
Chrome closing when exiting Firefox

### ISSUE THREAD

**Issue by tomelix** [andy-portmen/native-client#30]
I use "Open in chrome" addon in Firefox. When chrome is inactive and I open a page in chrome via the button in FF, then close FF, chrome gets also closed. When chrome is already active and I open a  new tab via the button in FF, then close FF, chrome stays opened. I would like chrome not to shut down when closing FF or to have an option to control this.

**Comment by andy-portmen** [comment]
Can you try the vice versa. This seems like a bug in Firefox (the way they handle external execution doesn't seem right)

**Comment by tomelix** [comment]
You mean "Open in Firefox" from Chrome? No issues with that.

**Comment by andy-portmen** [comment]
Yes "Open in Firefox" from Chrome browser. Regarding Firefox issue, I am not sure how to fix it! I'll keep this open for suggestions.
